src/TreeMenu/renderProps.tsx

```diff
@@ -6,9 +6,13 @@ import { Item } from './walk';
 const DEFAULT_PADDING = 0.75;
 const ICON_SIZE = 2;
 const LEVEL_SPACE = 1.75;
-const ToggleIcon = ({ on }: { on: boolean }) => (
-  <div role="img" aria-label="Toggle" className="toggle-icon-symbol">
-    {on ? '-' : '+'}
+const ToggleIcon = ({ on, openedIcon, closedIcon }: { on: boolean, openedIcon: any, closedIcon: any}) => (  
+  <div role="img" aria-label="Toggle" className="toggle-icon-symbol">    
+    {on ? 
+      openedIcon ? openedIcon()  : '-'
```

Comment:
The type of `openedIcon` and `closedIcon` should be more specific. `any` is not not encouraged in most cases. And it's a bit weird that you trigger them as function calls. I think it makes more sense if they are `ReactNode` type.

And instead of using nested ternaries, you can give default values to `openedIcon` and `closedIcon`.
Then you can simply do the following:
```
{on ? openedIcon : closedIcon }
```